.gitignore

```diff
@@ -43,6 +43,7 @@ reportJunit.xml
 # Visual Studio
 .vs/
 .config/dotnet-tools.json
+/system-tests.sln
```

Comment:
I guess you have C# dev kit :) 

![image](https://github.com/DataDog/system-tests/assets/11915659/71e0c3d4-fd2d-49fb-9eee-ae224ee37d9b)